README.md

```diff
@@ -1,41 +1,49 @@
 # BastCastle Simple Home Automation using Home Assistant
 
 ## The haHA Gateway Drug
```

Comment:
Not a fan of this personally, but I'm not really working on the readme so whatevs